### PR TITLE
PreReleaseView: Rename to EarlyAccessView and always show

### PR DESCRIPTION
**Pull request by cassidyjames**
Lots of little improvements, plus always show it.

### CODE REVIEW

**Comment by danrabbit** [comment]
I guess we probably don't want to show the "What's new" view if the only thing here is the early access warning

File: src/Views/EarlyAccessView.vala [hunk 1]
```diff
@@ -15,40 +15,52 @@
...
+        list_grid.add (style_label);
 
         var more_link = new Gtk.LinkButton.with_label (
             "https://github.com/orgs/elementary/projects/55",
-            _("View the 6.0 Release Project")
+            _("More issues on the 6.0 Release Project")
         ) {
             valign = Gtk.Align.END,
             vexpand = true
         };
```
Comment (cassidyjames) [inline src/Views/EarlyAccessView.vala:60]: I don't love how this is aligned now. Feedback welcome.

**Comment by danrabbit** [comment]
Screenshot from 2020-11-10 13 23 06@2x

Thoughts on badging the icon?

**Comment by danrabbit** [comment]
last rev

Screenshot from 2020-11-10 13 45 49@2x

**Review by danrabbit — APPROVED** [review]
LFG

**Pull request merged**